src/utils/local.ts

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line import/prefer-default-export
+export const isDevEnvironment = () =>
+  ['local'].includes(process.env.REACT_APP_APP_ENV || '');
```

Comment:
```suggestion
  ['local'].includes(process.env.REACT_APP_APP_ENV as string);
```

Have a look at the description in #649

Reply:
Sorry for all the individual comments. I'm getting new thoughts in real time.

If this was renamed to `isLocalEnvironment`, we can make it:
`process.env.REACT_APP_APP_ENV === 'local'`, right?

Reply:
Yes, but I think we might eventually want to use this in test as well